debug_toolbar/static/debug_toolbar/js/toolbar.js

```diff
@@ -111,20 +111,16 @@
                 var name = this.tagName.toLowerCase();
                 var ajax_data = {};
 
-                if (name == 'button') {
+                if (name === 'button' || name === 'a') {
                     var form = this.closest('form');
-                    ajax_data.url = this.getAttribute('formaction');
+                    ajax_data.url = name === 'button' ? this.getAttribute('formaction') : this.getAttribute('href');
```

Comment:
not sure if this has side-effects anywhere else. I could switch to using a button to view file however that did not looked well so switched back to using a link